auth/jwt_test.go

```diff
@@ -47,48 +50,134 @@ func (suite *JWTAuthenticatorTestSuite) createWrongToken(method jwt.SigningMetho
 	return token.SignedString([]byte(config.GetString("auth.jwt.secret")))
 }
 
-func (suite *JWTAuthenticatorTestSuite) TestAuthenticate() {
-	user := &TestUser{}
-	tokenAuthenticator := &JWTAuthenticator{}
+func (suite *JWTAuthenticatorTestSuite) TestGenerateToken() {
```

Comment:
```suggestion
func (suite *JWTAuthenticatorTestSuite) TestAuthenticate() {
```

I think it is better to name it that way because what we actually want to know is if that Authenticator is working properly. The `GenerateToken` function is tested more in-depth in other test functions in the suite.